src/commands/Utility/spotify.js

```diff
@@ -1,95 +1,132 @@
+const RichEmbed = require('discord.js').RichEmbed;
+const SpotifyWebApi = require('spotify-web-api-node');
+const SpotifyUriParser = require('spotify-uri');
 const got = require('got');
+const spotifyApi = new SpotifyWebApi();
+
+exports.run = async (bot, msg, args) => {
+    await got('https://spotify-auth.doxylamin.pw/').then(response => {
```

Comment:
One last quick change, this isn't quite how you use `await`. Also, I would put this in `exports.init`. Example:

```javascript
exports.init = async () => {
    const res = await got('https://spotify-auth.doxylamin.pw/');
    spotifyApi.setAccessToken(res.body);
};

exports.run = (bot, msg, args) => {
    // ...
};
```

This way it'll only run on bot-load, instead of every time the command is run.

Reply:
Can't do that, because the token is only valid for 1 hour, and - when another user uses the command the token created before gets invalid. So generating a new token for each request, is (more) fail-safe.

Reply:
Still, could you update this to properly use await? I mean, what you have is functional, but it’s a bit strange to use .then as well as await.